index.html

```diff
@@ -0,0 +1,151 @@
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  
+ <header>
+   <!--Header section-->
+    <nav class="topNav">
+        <a href="#"><i class="bi bi-facebook"></i></a>
+        <a href="#"><i class="bi bi-twitter"></i></a>
+        <a href="#">English</a>
+        <a href="#">My page</a>
+        <a href="#" class="signup">Logout</a>
+    </nav>
+    <!--Navigation mobile-->
+        <nav class="menu_mobile">
+                <div class="curtain">
+                    <a href="#" class="close">&times;</a>
+                    <div class="curtain-content">
+                            <a href="about.html">About</a>
+                            <a class="menu-link" href="#newsUpdate">updates</a>
+                            <a class="menu-link" href="#blog">Blog</a>
+                            <a class="menu-link" href="#">Report</a>
+                            <a class="menu-link" href="#">Contact Us</a>
+                    </div>
+                </div>
+
+                <i class="fa fa-bars open"></i>
+        </nav>
+    <!--Navigation desktop-->
+        <nav class="menu_desktop">
+          <div class="logo">
+            <img src="images/logo.png" alt="Logo-Image">
+            <a href="">Taas Homes</a>
+          </div>
+            <div class="nav_menu">
+              <a href="about.html">About</a>
+              <a class="menu-link" href="#newsUpdate">updates</a>
+              <a class="menu-link" href="#blog">Blog</a>
+              <a class="menu-link" href="#">Report</a>
+              <a class="menu-link" href="#">Contact Us</a>
+            </div>
+        </nav>
+</header>
+<!--Home section-->
+<section class="introduction">
+  <div class="content">
+    <h5>"Welcome to Taas Real Estate Paradise"</h5>
+  <h1>10 Units Left! Ready to move in 2&3 Beds</h1>
+  <p>Last chance to be part of the masterfully designed Moon Valley Apartments in Kileleshwa. Enjoy crazy discounts only for a limited time.
+    Specifically personalized for you! MORTGAGE ACCEPTED</p>
+  <h2>Visit our showroom in ICC Nairobi</h2>
+  <h3>@ KICC towers Nairobi CBD starting from 8.00am - 4.00pm</h3>
+  </div>
+</section>
+<!--Updates section--> 
+<section class="newsUpdate" id="newsUpdate">
+  <h2 class="sectionHeader">Ours News Update</h2>
+  <hr>
+  <div class="update-container">
+  <div class="update">
+      <ul>
+          <li>
+              <i class="car icon"></i>
+          </li>
+          <li class="headline"><h3>Parking</h3></li>
+          <li>We have ample parking space in all our highend properties in Keleleshwa, Kilimani and Eastlands. 
+          </li>
+      </ul>
+  </div>
+  <div class="update">
```

Comment:
kindly add one more element to this section , in the given design their are 5 elements but you have 4  👍 
we need to match the given design to our best 😄 
![image](https://user-images.githubusercontent.com/87448628/181750786-85fd2017-e2ae-4572-add0-051cc621268a.png)